app/controllers/cart_controller.rb

```diff
@@ -16,4 +16,10 @@ def create
 
     redirect_to photos_path
   end
+
+  def index
+    @cart_items = Cart.last.shopping_cart_items.all
+    @cart = Cart.last
```

Comment:
we will merge conflict here I guess, since we have been working in a session feature now